### PR TITLE
data release 2022-07-27

### PR DESCRIPTION
**Pull request by chejennifer**
frequent_2022_07_27_01_19_16
ipcc_2022_07_01_03_48_00
biomedical_2022_07_25_10_14_25
infrequent_2022_07_26_08_00_14

bigquery: datcom-store.dc_kg_2022_07_25_23_24_42

### CODE REVIEW

File: internal/server/ranking/ranking.go [hunk 1]
```diff
@@ -41,7 +41,7 @@ func s(str string) *string {
 // StatisticalVariable, where lower value means higher ranking.
 // Outer key is Import Name, inner key is Rank Key, value is score.
 var StatsRanking = map[string]map[RankKey]int{
-	"CensusPEP": {{MM: s("CensusPEPSurvey")}: 0}, // Population
+	"CensusPEP": {{MM: s("CensusPEPSurvey"), OP: s("P1Y")}: 0}, // Population
```
Comment (pradh) [inline internal/server/ranking/ranking.go:44]: I think we want: (CC @shifucun )

```
"USCensusPEP_Annual_Population": {{MM: s("CensusPEPSurvey"), OP: s("P1Y")}: 0},
"CensusPEP": {{MM: s("CensusPEPSurvey")}: 1},
```

e.g., search for "PEP" in https://autopush.datacommons.org/tools/statvar#Count_Person

Comment (shifucun) [reply internal/server/ranking/ranking.go:44]: Thanks, shanth, this is correct!

Comment (chejennifer) [reply internal/server/ranking/ranking.go]: updated

**Review by pradh — APPROVED** [review]
Thanks!

**Comment by chejennifer** [comment]
Thanks for reviewing!

**Pull request merged**